### PR TITLE
Remove TODO: add warning message together with the status

### PR DESCRIPTION
**Pull request by xichengliudui**
**Please provide a description of this PR:**

Remove TODO: add warning message together with the status

### CODE REVIEW

File: operator/pkg/helmreconciler/prune.go [hunk 1]
```diff
@@ -159,7 +159,6 @@ func (h *HelmReconciler) PruneControlPlaneByRevisionWithController(iopSpec *v1al
 			return errStatus,
 				fmt.Errorf("failed to check proxy infos: %v", err)
 		}
-		// TODO(richardwxn): add warning message together with the status
 		if len(pids) != 0 {
 			msg := fmt.Sprintf("there are proxies still pointing to the pruned control plane: %s.",
```
Comment (xichengliudui) [inline operator/pkg/helmreconciler/prune.go:163]: warning message have been added to status，line: 165

**Review by howardjohn — APPROVED** [review]

**Pull request merged**